src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;

```

Comment:
add `form-style: normal` as in checklist 
<img width="1017" alt="Screenshot 2022-12-20 at 2 48 58 PM" src="https://user-images.githubusercontent.com/60295799/208671237-1f4a6c74-4891-4f96-bf6d-d697931360f0.png">